easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-intel-2018.00-Python-3.6.2.eb

```diff
@@ -0,0 +1,20 @@
+name = 'SWIG'
+version = '3.0.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'intel', 'version': '2018.00'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
@reedts Please include a SHA256 checksum.

Reply:
@reedts Still missing SHA256 checksum, please add it?

Reply:
@boegel My bad, pushed to early. Here you go!